packages/api/package.json

```diff
@@ -28,7 +28,9 @@
 		"prepublishOnly": "yarn build"
 	},
 	"dependencies": {
+		"@types/psl": "^1.1.0",
```

Comment:
This feels wrong

Reply:
Yeah... well.. without it the thing is gonna fail for other users because they'd be missing the types.

Reply:
No... skipLibCheck is a thing

Reply:
And since this module isn't exposed nor used on any of our types, it doesn't matter if it's not installed.